.buildkite/scripts/compile.sh

```diff
@@ -2,4 +2,3 @@
 set -e
 
 yarn install --frozen-lockfile
-yarn tsc
```

Comment:
If we're removing `yarn tsc` here, does this buildkite step even need to exist?

Or do we need to replace it with something else?